### PR TITLE
Mariocube.BRRESViewer version 1.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103139)

### CODE REVIEW

**Comment by stephengillie** [comment]
The PackageIdentifier is "BRLYTEditor" but the EXE and the package say BRRES Viewer. 
image

**Comment by upintheairsheep** [comment]
> The PackageIdentifier is "BRLYTEditor" but the ZIP, EXE and Package Name, and program title say "BRRES Viewer". image
> 
> Should the PackageIdentifier be updated to "BRRESViewer"?

I don't have a system to rename these folders. I only use iOS (which I hate) and I contribute via a command-line-only VPS

**Comment by stephengillie** [comment]
> I don't have a system to rename these folders. I only use iOS (which I hate) and I contribute via a command-line-only VPS

Oh... iOS was built on something like Unix, and uses common commands. Would need:
- Git commands to switch to a given branch
- Linux commands to manage files
- Git commands to commit these changes

**Comment by upintheairsheep** [comment]
> > I don't have a system to rename these folders. I only use iOS (which I hate) and I contribute via a command-line-only VPS
> 
> Oh... iOS was built on something like Unix, and uses common commands. Would need:
> 
> * Git commands to switch to a given branch
> * Linux commands to manage files
> * Git commands to commit these changes

We should integrate moving stuff into WinGetBot as a command

**Comment by upintheairsheep** [comment]
> > I don't have a system to rename these folders. I only use iOS (which I hate) and I contribute via a command-line-only VPS
> 
> Oh... iOS was built on something like Unix, and uses common commands. Would need:
> 
> * Git commands to switch to a given branch
> * Linux commands to manage files
> * Git commands to commit these changes

We should integrate moving stuff into WinGetBot as a command

File: manifests/m/Mariocube/BRRESViewer/1.0/Mariocube.BRRESViewer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.BRLYTEditor
```
Comment (stephengillie) [inline manifests/m/Mariocube/BRRESViewer/1.0/Mariocube.BRRESViewer.installer.yaml:4]:
```suggestion
PackageIdentifier: Mariocube.BRRESViewer
```

File: manifests/m/Mariocube/BRRESViewer/1.0/Mariocube.BRRESViewer.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.BRLYTEditor
```
Comment (stephengillie) [inline manifests/m/Mariocube/BRRESViewer/1.0/Mariocube.BRRESViewer.yaml:4]:
```suggestion
PackageIdentifier: Mariocube.BRRESViewer
```

File: manifests/m/Mariocube/BRRESViewer/1.0/Mariocube.BRRESViewer.installer.yaml (hunk 1 above)
Comment (stephengillie) [inline manifests/m/Mariocube/BRRESViewer/1.0/Mariocube.BRRESViewer.installer.yaml:4]:
```suggestion
PackageIdentifier: Mariocube.BRRESViewer
```

File: manifests/m/Mariocube/BRRESViewer/1.0/Mariocube.BRRESViewer.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.BRLYTEditor
```
Comment (stephengillie) [inline manifests/m/Mariocube/BRRESViewer/1.0/Mariocube.BRRESViewer.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Mariocube.BRRESViewer
```

File: manifests/m/Mariocube/BRRESViewer/1.0/Mariocube.BRRESViewer.yaml (hunk 2 above)
Comment (stephengillie) [inline manifests/m/Mariocube/BRRESViewer/1.0/Mariocube.BRRESViewer.yaml:4]:
```suggestion
PackageIdentifier: Mariocube.BRRESViewer
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**